config/initializers/mail.rb

```diff
@@ -0,0 +1,13 @@
+require 'mail'
+
+options = { :address              => $app_config.mail.smtp_server,
```

Comment:
can you please move key&value to next line. Also please use new hash syntax without rockets.